pression/compressor/compressorFastLZ.cpp

```diff
@@ -47,7 +47,7 @@ static bool _register()
     return true;
 }
 
-static const bool _initialized = _register();
+static const bool LB_UNUSED _initialized = _register();
```

Comment:
Static is not needed inside an anonymous namespace. You can also remove it in the function declarations up.
Are you sure the warning is not spurious in gcc 5.4 and the compiler is not optimizing out the call?